apps/acrs/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const SummaryPageBehaviour = require('hof').components.summary;
-const SaveFormSession = require('./behaviours/save-form-session');
+const ResumeSession = require('./behaviours/resume-form-session');
```

Comment:
Is it somehow getting rid of `SaveFormSession here`? Think it makes it change not add the new behaviour

Reply:
Not getting rid of it SaveFormSession will be used definitely

Reply:
I'm saying you are deleting it here.

Reply:
We agreed to remove as it will fail tests as it was